cmake/FindESMF.cmake

```diff
@@ -0,0 +1,47 @@
+
+if (DEFINED ENV{ESMFMKFILE})
```

Comment:
UFS has FindESMF.cmake  FindNetCDF.cmake  FindPIO.cmake under  CMakeModules/Modules, I assume ufs does not use the cmake here from CMEPS. Are these cmake related files used in CIME?

Reply:
Yes. There has been some confusion on my part for what is required for cime vs our build but I believe this to be good for both sides.